cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,247 +3,378 @@
    Bureau structure page at /the-bureau/bureau-structure/.
    ========================================================================== */
 
+@margin__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@spine-width__px: 5px;
+@spine-color: @gray-40;
+@node-width__desktop__px: 405px;
```

Comment:
If we don't specify a `_px` suffix, these could be updated to other units (like %) without updating the variable. It also makes a lot more sense for the `none` one below.